src/blog/2023/08/UNS-article.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Why the Automation Pyramid blocks digital transformation - The Role of Unified Namespace
+subtitle: A Critical Examination of the Automation Pyramid's Obstruction to Digital Transformation
+description: This article analyzes the Automation Pyramid's constraints and explains the Unified Namespace, highlighting its potential to evolve digital transformation in manufacturing.
+date: 2023-08-08 
+authors: ["marian-demme"]
+image: blog/2023/08/images/UNS/UNS-Headline.svg
+tags:
+    - posts
+---
+A few years ago, I wrote an [article](https://www.linkedin.com/pulse/iiot-circle-marian-raphael-demme/), in German, detailing my understanding of how the Automation Pyramid, a widely adopted reference model for the IT landscape of manufacturing firms, is essentially hindering digital transformation. Now, as conversations around the Unified Namespace (UNS) and particular frameworks continue to evolve, I revisit my earlier notions, review the latest updates to reference frameworks, and update my article.
+<!--more-->
+## The Pyramid’s Dilemma
```

Comment:
this article makes many references to the automation pyramid, especially in this section.  Yet I do not see an actual picture of the automation pyramid.  Somewhere in this paragraph would be a good place to show the actual automation pyramid.  There's many variants of the pyramid, but I don't think it matters which, just that you show the reader a picture of an actual pyramid.

Reply:
[Added Automation Pyramid - Visualization](https://github.com/flowforge/website/pull/1009/commits/7e243e0a2a063a668375fce02a3170d81f1be090)